src/handlers/transfers/handler.js

```diff
@@ -197,88 +198,90 @@ const processFulfilMessage = async (message, functionality, span) => {
-
-    /**
-     * TODO: BULK-Handle at BulkProcessingHandler (not in scope of #967)
-     * HOWTO: For regular transfers, send the fulfil from non-payee dfsp.
-     * Not sure if it will apply to bulk, as it could/should be captured
-     * at BulkPrepareHander. To be verified as part of future story.
-     */
-
-    // Publish message to Position Handler
-    // Key position abort with payer account id
-    const payerAccount = await Participant.getAccountByNameAndCurrency(transfer.payerFsp, transfer.currency, Enum.Accounts.LedgerAccountType.POSITION)
-    await Kafka.proceed(Config.KAFKA_CONFIG, params, { consumerCommit, fspiopError: apiFSPIOPError, eventDetail, fromSwitch, toDestination: transfer.payerFsp, messageKey: payerAccount.participantCurrencyId.toString(), hubName: Config.HUB_NAME })
-
-    /**
-     * Send patch notification callback to original payee fsp if they asked for a a patch response.
-     */
-    if (action === TransferEventAction.RESERVE) {
-      Logger.isInfoEnabled && Logger.info(Util.breadcrumb(location, `callbackReservedAborted--${actionLetter}3`))
-
-      // Set the event details to map to an RESERVE_ABORTED event targeted to the Notification Handler
-      const reserveAbortedEventDetail = { functionality: TransferEventType.NOTIFICATION, action: TransferEventAction.RESERVED_ABORTED }
-
-      // Extract error information
-      const errorCode = apiFSPIOPError && apiFSPIOPError.errorInformation && apiFSPIOPError.errorInformation.errorCode
-      const errorDescription = apiFSPIOPError && apiFSPIOPError.errorInformation && apiFSPIOPError.errorInformation.errorDescription
-
-      // TODO: This should be handled by a PATCH /transfers/{id}/error callback in the future FSPIOP v1.2 specification, and instead we should just send the FSPIOP-Error instead! Ref: https://github.com/mojaloop/mojaloop-specification/issues/106.
-      const reservedAbortedPayload = {
-        transferId: transferAbortResult && transferAbortResult.id,
-        completedTimestamp: transferAbortResult && transferAbortResult.completedTimestamp && (new Date(Date.parse(transferAbortResult.completedTimestamp))).toISOString(),
-        transferState: TransferState.ABORTED,
-        extensionList: { // lets add the extension list to handle the limitation of the FSPIOP v1.1 specification by adding the error cause...
-          extension: [
-            {
-              key: 'cause',
-              value: `${errorCode}: ${errorDescription}`
-            }
-          ]
-        }
-      }
-      message.value.content.payload = reservedAbortedPayload
-      await Kafka.proceed(Config.KAFKA_CONFIG, params, { consumerCommit, eventDetail: reserveAbortedEventDetail, fromSwitch: true, toDestination: transfer.payeeFsp, hubName: Config.HUB_NAME })
-    }
-
-    throw apiFSPIOPError
   }
+  // Lets validate FSPIOP Source & Destination Headers
```

Comment:
Why not use the headers to check if an fsp is inScheme and disabling these only if one is not in the scheme.
Feel like disabling these validatons entirely will break something in GP.

Reply:
Thanks @kleyow addressed your concern.